distribution/solr-distro/pom.xml

```diff
@@ -101,11 +101,11 @@
                         </goals>
                         <configuration>
                             <url>
-                                http://artifacts.codice.org/content/repositories/thirdparty/org/apache/archive/solr/7.4.0/solr-7.4.0.zip
+                                http://artifacts.codice.org/content/repositories/thirdparty/org/apache/archive/solr/${solr.version}/solr-${solr.version}.zip
                             </url>
                             <unpack>true</unpack>
                             <failOnError>true</failOnError>
-                            <sha1>f63157ae606a081a48513d5d9ae0a5d0d9741ce2</sha1>
+                            <sha512>aca27ff6942872a571160f49f5e4d43090dbe9bb37da0d1999557e1655f25055173ef999bfb8eba8f19f19b9c438b43205c805d0d294b9756cf5c0aaf66a656a</sha512>
```

Comment:
You can verify the SHA here: https://www.apache.org/dist/lucene/solr/7.7.2/solr-7.7.2.zip.sha512